feature-libs/checkout/base/components/checkout-review/checkout-review-overview/checkout-review-overview.component.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Component } from '@angular/core';
+import { ActiveCartFacade, Cart } from '@spartacus/cart/base/root';
+import { Observable } from 'rxjs';
+
+@Component({
+  selector: 'cx-checkout-review-overview',
+  templateUrl: './checkout-review-overview.component.html',
```

Comment:
can you add this `changeDetection: ChangeDetectionStrategy.OnPush,`? also in the `review-shipping` component.